app/transformations/LandOrPropertyTransactionsTransformer.scala

```diff
@@ -548,7 +548,7 @@ class LandOrPropertyTransactionsTransformer @Inject() extends Transformer {
                       identityType = receivedLandType,
                       idNumber = None,
                       reasonNoIdNumber = None,
-                      otherDescription = Some(other.description)
+                      otherDescription = if (other.description.nonEmpty) Some(other.description) else None
```

Comment:
Could be rewritten as 
`
Option.when(other.description.nonEmpty)(other.description)
`